docs/learning/training/markdown-guide.md

```diff
@@ -63,28 +55,83 @@ Commonly used formatting options include the following:
 | `---`                                        | <hr>                                       |
 | `[Link Name](https://www.markdownguide.org)` | [Link Name](https://www.markdownguide.org) |
 
----
+### Diagrams
+
+You can generate diagrams and flowcharts from text by using [Mermaid](https://mermaid-js.github.io/mermaid/#/).
+
+The [Mermaid Live Editor](https://mermaid-js.github.io/mermaid-live-editor/) helps you learn Mermaid and debug
+issues in your Mermaid code. Use it to identify and resolve issues in your diagrams.
+
+To generate a diagram or flowchart, write your text inside the mermaid block:
+
+````markdown
+```mermaid
+graph TD
+  A[Dinner]-->B[Tacos]
+  A-->C[Noodles]
+  B-->D[Fulfillment]
+  C-->D
+```
+````
+
+```mermaid
+graph TD
+  A[Dinner]-->B[Tacos]
+  A-->C[Noodles]
+  B-->D[Fulfillment]
+  C-->D
+```
+
+#### Collapse
+
+A collapsed content section is used to hide information until a user chooses to reveal it with a click or tap on the summary text.
+The hidden content is revealed inline. For example, this code:
+
+```markdown
+<details>
+<summary markdown="span">This is the summary text, click me to expand</summary>
+
+This is the detailed text.
+
+You can learn more about expected usage of this approach in the [ThothTech markdown guide](https://github.com/thoth-tech/handbook/blob/main/docs/learning/training/markdown-guide.md).
+
+</details>
+```
+
+results in:
+
+<details>
+<summary markdown="span">This is the summary text, click me to expand</summary>
+
+This is the detailed text.
+
+You can learn more about expected usage of this approach in the [ThothTech markdown guide](https://github.com/thoth-tech/handbook/blob/main/docs/learning/training/markdown-guide.md).
+
+</details>
+
+To add markdown in the collapsed heading, be sure to add:
```

Comment:
what do you mean by "add markdown in the collapsed heading"?  Maybe there is a different way to word it?

Reply:
Sorry for the confusion @talzid. I was under the impression that markdown content can't be rendered inside the `<details>` block. Apparently, the following text,

```mardown
<details>

### Manificient header
</details>
```
can be rendered as 

<details>

### Manificient header
</details>

I will remove this line 👍🏼